src/content/byoc/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Okteto Bring Your Own Cloud (BYOC)
+description: Introduction to Okteto Bring Your Own Cloud (BYOC)
+---
+
+Okteto’s Bring Your Own Cloud (BYOC) offering allows you to run Okteto on your own cloud infrastructure while still enjoying the benefits of a fully managed experience.
+
+This is ideal for teams who need to meet strict security or compliance requirements, require custom infrastructure configurations, or simply want to maintain full control over their environments with a SaaS-like experience.
+
+We currently support BYOC on:
+
+- [**Amazon Web Services (AWS)**](byoc/aws/index.mdx)
+- [**Google Cloud Platform (GCP)**](byoc/gcp/index.mdx)
+
+## How the BYOC Model Works
+
+With BYOC, you bring the cloud provider account; we bring the platform and operational expertise.
+
+- You provision and secure the cloud account
+- Okteto creates the Kubernetes cluster and installs all necessary components
```

Comment:
We create the kubernetes cluster and othe infrastructure (vpcs, buckets, accounts, etc). Maybe we can instead say something like this?
```suggestion
- Okteto creates the all the necessary cloud infrastructure and software components
```